CHANGELOG.md

```diff
@@ -4,8 +4,9 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
-## [1.0.2] - 2018-11-06
+## [1.0.3] - 2018-11-07
```

Comment:
This needs to be updated for the new version.

Reply:
Updated. Thanks